docs/02-connect/01-get_a_wallet.md

```diff
@@ -4,14 +4,398 @@ title: Get a Wallet
+</p>
+
+3. The **Importing wallet for TokenMint Chain** page appears. Enter the wallet name and password. Click **Next**. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-import-walletname.png" alt="Cobalt Import WalletName" width="300" height="200" />
+</p>
+
+4. The **Enter seed** page appears. Enter your wallet’s 24-words seed phrase. Click **Confirm**. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-importwallet-seed1.png" alt="Cobalt Wallet Seed" width="300" height="200" />
+</p>
+
+5. When the import is complete, the Cobalt wallet displays the restored wallet.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-importwallet-complete.png" alt="Cobalt Wallet Complete" width="300" height="200" />
+</p>
+
+### Account Settings
+
+The Cobalt wallet provides a setting feature where you can manage your account. 
+
+Click on the the **Account icon** on the top-right corner of the page to display the Account menu:
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-account.png" alt="Cobalt Account" width="300" height="200" />
+</p>
+
+To manage your Cobalt wallet, use the Account settings to perform the following options:
+
+* **Security** - view your private key or recovery seed phrase
+* **Lock** - lock your wallet
+* **Connect Sites** - display a list of connected sites
+* **Manage Accounts** - rename your wallet name, hide your wallet from view, or add a new account
+* **Manage Networks** - delete networks that you configured or add a custom network
+* **Terms of Use** - display the Terms of Use page
+
+#### Security
+The Cobalt wallet allows you to view and copy your wallet’s private key. You need the wallet’s private key if you change your web browser or computer. You can also view the recovery seed phrase of your wallet. 
+
+1. In the **Account** menu, click **Security**. The **Security** page appears. 
+
+<p>
+<img src="/img/docs/get-started/cobalteon-account-security.png" alt="Cobalt Security" width="300" height="200" />
+</p>
+
+##### View Private Key
+
+2. Click **View Private Key**. The private key is used to access your account. It is highly recommended that you write down your private key and keep it in a safe place. Enter your password to access your private key. Click **Next**.
+
+<p>
+<img src="/img/docs/get-started/gobie-cobalteon-account-privatekey.png" alt="Cobalt PrivateKey" width="300" height="200" />
+</p>
+
+##### View Recovery Seed Phrase
+
+3. In the **Security** page, click **View Recovery Seed Phrase**. You will need your wallet’s seed phrase if you change browser or computer. The **View Recovery Seed Phase** page appears. 
+
+
+<p>
+<img src="/img/docs/get-started/cobalteon-account-recovery.png" alt="Cobalt PrivateKey" width="300" height="200" />
+</p>
+
+4. The **View Recovery Seed Phase** page displays your secret recovery phrase. Write it down and keep it safe. Click **Close**.
+
+<p>
+<img src="/img/docs/get-started/cobalteon-account-seedphrase.png" alt="Cobalt SeedPhrase" width="300" height="200" />

```

Comment:
If possible, I would blur the seed phrase in all screenshots

Reply:
I tried to blur the seed phrase section of the screenshot with the image tool online but did not work since the tool is limited in capability...it only blurs out the entire screenshot and not selected portions. I will continue to research other tools that can provide blurring selected portions of an image.